modules/pel/peltool/peltool.py

```diff
@@ -626,7 +662,7 @@ def main():
         config.severities.extend(severityGroupValues[sev] for sev in args.severities)
 
     if args.file:
-        parseAndPrintPELFile(args.file, config, True)
+        parseAndPrintPELFile(args.file, config, True, args.hex)
```

Comment:
+1

Reply:
Thats right. I followed args.rev method. I missed to check on config. config is much easier. I will make one more commit in future to remove args.rev. so, no body will be deviated because of this approach.